### PR TITLE
feat(sca): Support the new --image flag along the --docker-image flag

### PR DESCRIPTION
**Pull request by reemodo**
## Description

I pushed a PR to Support the new --image flag along with the --docker-image flag. 

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my feature, policy, or fix is effective and works
- [ ] New and existing tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by nimrodkor — APPROVED** [review]

**Comment by Saarett** [comment]
@reemodo Great job!
1. Please update the README in the appropriate places, feel free to talk with me if you're not sure where
2. In the main.py file there are a couple of places associated with the `docker-image` flag, please update them as well

**Review by itai1357 — APPROVED** [review]

**Review by itai1357 — COMMENTED** [review]
make sure to try to run one time with the new flag. I think it should work

**Comment by reemodo** [comment]
> 

Yeah, sure. I tried it, and it works.

File: checkov/main.py [hunk 1]
```diff
@@ -144,7 +145,10 @@ def run(banner: str = checkov_banner, argv: list[str] = sys.argv[1:]) -> int | N
         "python_executable": sys.executable,
         "python_version": sys.version,
         "checkov_executable": sys.argv[0],
-        "args": parser.format_values(sanitize=True).split('\n')
+        "args": parser.format_values(sanitize=True).split('\n'),
+        "OS_system_info": platform.platform(),
+        "CPU_architecture": platform.processor(),
+        "Python_implementation":platform.python_implementation()
```
Comment (Saarett) [inline checkov/main.py:151]: @reemodo How is this related to this PR?

File: checkov/main.py [hunk 2]
```diff
@@ -572,7 +576,11 @@ def run(self, banner: str = checkov_banner) -> int | None:
             "python_executable": sys.executable,
             "python_version": sys.version,
             "checkov_executable": sys.argv[0],
-            "args": self.parser.format_values(sanitize=True).split('\n')
+            "args": self.parser.format_values(sanitize=True).split('\n'),
+            "OS_system_info": platform.platform(),
+            "CPU_architecture": platform.processor(),
+            "Python_implementation":platform.python_implementation()
```
Comment (Saarett) [inline checkov/main.py:582]: Same goes here

**Review by Saarett — CHANGES_REQUESTED** [review]
@reemodo Please take a look at my latest comments before merging
